absl/strings/numbers.h

```diff
@@ -25,8 +25,12 @@
 #define ABSL_STRINGS_NUMBERS_H_
 
 #ifdef __SSE4_2__
```

Comment:
It doesn't look like `__SSE4_2__` is defined by MSVC, so I don't think this is the correct fix. Are you trying to define `__SSE4_2__` yourself under MSVC?